<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Esther Plomp">
<meta name="dcterms.date" content="2022-09-04">

<title>TNW-OS-support - Metadata standards and repositories for Applied Sciences/TNW</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">TNW-OS-support</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../about.html">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/EstherPlomp/TNW-OS-support"><i class="bi bi-github" role="img">
</i> 
 <span class="menu-text"></span></a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Metadata standards and repositories for Applied Sciences/TNW</h1>
                                <div class="quarto-categories">
                <div class="quarto-category">Open Data</div>
                <div class="quarto-category">Research Data Management</div>
                <div class="quarto-category">FAIR</div>
              </div>
                  </div>
  </div>
    
  
  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Esther Plomp </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">September 4, 2022</p>
      </div>
    </div>
    
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content">




<section id="relevant-metadata-standards" class="level2">
<h2 class="anchored" data-anchor-id="relevant-metadata-standards">Relevant metadata standards</h2>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/isa-tab.html">ISA-Tab</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/mibbi-minimum-information-biological-and-biomedical-investigations.html">MIBBI (Minimum Information for Biological and Biomedical Investigations)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/nexus.html">NeXus</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/ome-xml-open-microscopy-environment-xml.html">OME-XML (Open Microscopy Environment XML)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/pdbx-mmcif.html">PDBx/mmCIF (Protein Data Bank Exchange Dictionary and the Macromolecular Crystallographic Information Framework)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cif-crystallographic-information-framework.html">CIF (Crystallographic Information Framework)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/qudex-qualitative-data-exchange-format.html">QuDEx (Qualitative Data Exchange Format)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/sdmx-statistical-data-and-metadata-exchange.html">SDMX (Statistical Data and Metadata Exchange)</a></p>
</section>
<section id="potentially-relevant-metadata-standards" class="level2">
<h2 class="anchored" data-anchor-id="potentially-relevant-metadata-standards">Potentially relevant metadata standards</h2>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/genome-metadata.html">Genome Metadata</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cf-climate-and-forecast-metadata-conventions.html">CF (Climate and Forecast) Metadata Conventions</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cim-common-information-model.html">CIM (Common Information Model)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/csmd-cclrc-core-scientific-metadata-model.html">CSMD (Core Scientific Metadata Model)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/dif-directory-interchange-format.html">DIF (Directory Interchange Format)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/iso-19115.html">ISO 19115</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/observations-and-measurements.html">Observations and Measurements</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cerif.html">CERIF (Common European Research Information Format)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/data-package.html">Data Package</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/abcd-access-biological-collection-data.html">ABCD (Access to Biological Collection Data)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/mods.html">MODS (Metadata Object Description Schema)</a></p>
<p><a href="https://github.com/edamontology/edam-bioimaging/blob/master/README.md">EDAM-bioimaging</a></p>
</section>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section"></h2>
</section>
<section id="repositories" class="level2">
<h2 class="anchored" data-anchor-id="repositories">Repositories</h2>
<p><a href="http://www.ddbj.nig.ac.jp/">DNA DataBank of Japan (DDBJ)</a></p>
<p><a href="http://www.ebi.ac.uk/ena/">European Nucleotide Archive (ENA)</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/snp">dbSNP</a></p>
<p><a href="https://www.ebi.ac.uk/eva/"><strong>European Variation Archive (EVA)</strong></a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/dbvar/">dbVar</a></p>
<p><a href="http://www.ebi.ac.uk/dgva/">Database of Genomic Variants Archive (DGVa)</a></p>
<p><a href="http://www.ebi.ac.uk/metagenomics">EBI Metagenomics</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/Traces/home/">NCBI Trace Archive</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/sra"><strong>NCBI Sequence Read Archive (SRA)</strong></a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/assembly">NCBI Assembly</a></p>
<p><a href="http://data.sbgrid.org/">Structural Biology Data Grid</a></p>
<p><a href="http://wwpdb.org/">Worldwide Protein Data Bank (wwPDB)</a></p>
<p><a href="http://www.uniprot.org">UniProtKB</a></p>
<p><a href="http://www.bmrb.wisc.edu/">Biological Magnetic Resonance Data Bank (BMRB)</a></p>
</section>
<section id="rna-seqqpcr" class="level2">
<h2 class="anchored" data-anchor-id="rna-seqqpcr">RNA-seq/qpcr</h2>
<ul>
<li><p><a href="https://www.ebi.ac.uk/ena/browser/home">European Nucleotide Archive</a> (ENA)</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/sra">Sequence Read Archive</a> (SRA)</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus</a> (GEO)</p></li>
<li><p><a href="https://www.encodeproject.org/">ENCODE</a></p></li>
<li><p><a href="https://exrna-atlas.org/">exRNA Atlas</a></p></li>
<li><p><a href="https://rdml.org/miqe.html">Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE)</a></p></li>
</ul>
</section>
<section id="microscopy" class="level2">
<h2 class="anchored" data-anchor-id="microscopy">Microscopy</h2>
<ul>
<li><p><a href="https://idr.openmicroscopy.org/"><strong>IDR</strong></a></p></li>
<li><p><a href="http://www.emdatabank.org/">Electron Microscopy Data Bank (EMDB)</a></p></li>
<li><p><a href="https://www.ebi.ac.uk/pdbe/emdb/empiar/"><strong>EMPIAR</strong></a></p></li>
<li><p><a href="http://www.cellimagelibrary.org/">Cell Image Library</a></p></li>
<li><p><a href="https://www.ebi.ac.uk/bioimage-archive/">BioImage Archive</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41592-021-01166-8">REMBI: Recommended Metadata for Biological Images</a></p></li>
<li><p><a href="https://grussmayerlab.github.io/open-blink/about/">Open blink project</a></p></li>
</ul>
</section>
<section id="tools" class="level2">
<h2 class="anchored" data-anchor-id="tools">Tools</h2>
<p><a href="https://www.iochem-bd.org/">ioChem-BD</a></p>
</section>
<section id="simulation-outputs" class="level2">
<h2 class="anchored" data-anchor-id="simulation-outputs">Simulation outputs</h2>
<ul>
<li><p><a href="https://sed-ml.org/">SED-ML</a></p></li>
<li><p><a href="http://biomodels.net/miase">Minimum Information About a Simulation Experiment (MIASE)</a></p></li>
</ul>
</section>
<section id="open-research" class="level2">
<h2 class="anchored" data-anchor-id="open-research">Open Research</h2>
<p><a href="https://osf.io/3r8hb/">Open Research: Examples of good practice, and resources across disciplines</a></p>
</section>
<section id="other" class="level2">
<h2 class="anchored" data-anchor-id="other">Other</h2>
<ul>
<li><a href="https://www.youtube.com/watch?v=o1r4x4qVEBM">Dr.&nbsp;Christoph Steinbeck on “Informatics, Datasets and the Curation of the Future”</a></li>
</ul>
</section>
<section id="mri-data" class="level2">
<h2 class="anchored" data-anchor-id="mri-data">MRI data</h2>
<ul>
<li><p><a href="https://doi.org/10.5281/zenodo.4740126">Brain MRI data sharing guide</a></p></li>
<li><p><a href="https://doi.org/10.1002/hbm.25351">The Open Brain Consent</a></p></li>
</ul>
</section>
<section id="image-data" class="level2">
<h2 class="anchored" data-anchor-id="image-data">Image Data</h2>
<ul>
<li><p><a href="https://central.xnat.org/">XNAT Central</a> (neuroimaging, but also from oncology, orthopaedics and cardiology)</p></li>
<li><p><a href="https://www.cancerimagingarchive.net/">The Cancer Imaging Archive</a></p></li>
</ul>
</section>
<section id="publications" class="level2">
<h2 class="anchored" data-anchor-id="publications">Publications</h2>
<ul>
<li><a href="https://doi.org/10.1016/j.dche.2022.100072">Theisen et al.&nbsp;2023</a></li>
</ul>
</section>
<section id="battery" class="level2">
<h2 class="anchored" data-anchor-id="battery">Battery</h2>
<p>[..E]arly independent battery-data activities include those of <a href="https://batteryarchive.org/">Battery Archive</a>, BIG-MAP [(<a href="https://www.big-map.eu/dissemination/battinfo">BattINFO ontology</a>)], <a href="https://batterieseurope.eu/">Batteries Europe,</a> and the <a href="https://www.faraday.ac.uk/">Faraday Institution</a>. Their diversity of locations and formats underscores the critical need for a singular approach to improve uniformity.38,41 - <a href="https://doi.org/10.1016/j.joule.%202022.08.008">Ward et al.&nbsp;2022</a></p>
<ul>
<li><p><a href="https://doi.org/10.1016/j.softx.2020.100506">BEEP</a>: A Python library for Battery Evaluation and Early Prediction</p></li>
<li><p><a href="https://github.com/Battery-Intelligence-Lab/galvanalyser">Galvanalyser</a> is a system for automatically storing data generated by battery cycling machines in a database</p></li>
<li><p><a href="nrel.gov/transportation/microstructure.html">Battery Microstructures Library</a></p></li>
<li><p><a href="https://www.nrel.gov/transportation/matbox.html">MATBOX</a>: Microstructure Analysis Toolbox (Li-ion batteries)</p></li>
<li><p><a href="https://made.ee.ethz.ch/research/open-source-data-and-software/bruggemanestimator.html">BruggemanEstimator</a>: Open Source Code for Estimating the Tortuosity Estimation of Lithium Ion Battery Porous Electrodes from SEM images</p></li>
<li><p><a href="https://github.com/tldr-group/taufactor">TauFactor</a> is an application for calculating tortuosity factors from tomographic data.</p></li>
<li><p><a href="https://github.com/materials-data-facility/battery-data-toolkit">Battery Data Toolkit</a>, converts battery testing data from native formats to a standardized HDF5 file.</p></li>
<li><p><a href="https://doi.org/10.1002/aenm.202102702">Ontologies for battery data</a> (<a href="https://www.big-map.eu/dissemination/battinfo">BattINFO</a> &amp; BVCO)</p>
<ul>
<li>“There are currently two major ongoing initiatives dedicated to ontologizing the battery domain: The Battery Interface Ontology (BattINFO) and the Battery Value Chain Ontology (BVCO). BattINFO describes batteries on the cell level and below, including not only components, materials, and their interfaces, but also electrochemical processes, models, and characterization data. The objective of BattINFO is to support AI workflows and interoperability of battery data in the research and development community. On the other hand, BVCO describes aspects of the battery value chain with a strong focus on battery manufacturing and recycling. Both BattINFO and BVCO stem from the top-level ontology EMMO and are publicly available under open-source licenses.”</li>
</ul></li>
<li><p><a href="https://www.doi.org/10.1002/batt.202100117">BIG-MAP Data Management Plan article</a> + <a href="https://www.big-map.eu/dissemination/public-project-deliverables">deliverable</a></p></li>
<li><p><strong>Repositories:</strong></p>
<ul>
<li><p><a href="https://icsd.products.fiz-karlsruhe.de/">Inorganic Crystal Structure Database</a> (ICSD)</p></li>
<li><p><a href="https://www.materialscloud.org/">Materials Cloud</a></p></li>
<li><p><a href="https://materialsproject.org/">Materials Project</a></p></li>
<li><p><a href="https://cmr.fysik.dtu.dk/">Computational Materials Repository</a></p></li>
</ul></li>
</ul>
</section>
<section id="bionano" class="level2">
<h2 class="anchored" data-anchor-id="bionano">Bio–nano</h2>
<p><a href="https://doi.org/10.1038/s41565-018-0246-4">Minimum information reporting in bio–nano experimental literature</a> (MIRIBEL)</p>
</section>
<section id="more-information" class="level2">
<h2 class="anchored" data-anchor-id="more-information">More information</h2>
<ul>
<li><p><a href="https://rdamsc.bath.ac.uk/subject-index">Metadata Standards Catalog</a></p></li>
<li><p><a href="https://fairsharing.org/">FAIRsharing.org</a></p></li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>