<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Esther Plomp">
<meta name="dcterms.date" content="2022-09-04">

<title>Metadata standards and repositories for Applied Sciences/TNW – TNW-OS-support</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">TNW-OS-support</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/EstherPlomp/TNW-OS-support"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Metadata standards and repositories for Applied Sciences/TNW</h1>
                                <div class="quarto-categories">
                <div class="quarto-category">Open Data</div>
                <div class="quarto-category">Research Data Management</div>
                <div class="quarto-category">FAIR</div>
              </div>
                  </div>
  </div>
    
  
  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Esther Plomp </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">September 4, 2022</p>
      </div>
    </div>
    
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content">





<section id="relevant-metadata-standards" class="level2">
<h2 class="anchored" data-anchor-id="relevant-metadata-standards">Relevant metadata standards</h2>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/isa-tab.html">ISA-Tab</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/mibbi-minimum-information-biological-and-biomedical-investigations.html">MIBBI (Minimum Information for Biological and Biomedical Investigations)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/nexus.html">NeXus</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/ome-xml-open-microscopy-environment-xml.html">OME-XML (Open Microscopy Environment XML)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/pdbx-mmcif.html">PDBx/mmCIF (Protein Data Bank Exchange Dictionary and the Macromolecular Crystallographic Information Framework)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cif-crystallographic-information-framework.html">CIF (Crystallographic Information Framework)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/qudex-qualitative-data-exchange-format.html">QuDEx (Qualitative Data Exchange Format)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/sdmx-statistical-data-and-metadata-exchange.html">SDMX (Statistical Data and Metadata Exchange)</a></p>
<p><a href="https://sbolstandard.org/">The Synthetic Biology Open Language (SBOL)</a> &amp; <a href="https://sbolstandard.org/visual-about/">SBOL Visual</a></p>
<p><a href="https://www.beilstein-institut.de/en/projects/strenda/guidelines/">STRENDA</a></p>
<p><a href="https://mlcommons.org/2024/03/croissant_metadata_announce/">Croissant</a></p>
</section>
<section id="potentially-relevant-metadata-standards" class="level2">
<h2 class="anchored" data-anchor-id="potentially-relevant-metadata-standards">Potentially relevant metadata standards</h2>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/genome-metadata.html">Genome Metadata</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cf-climate-and-forecast-metadata-conventions.html">CF (Climate and Forecast) Metadata Conventions</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cim-common-information-model.html">CIM (Common Information Model)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/csmd-cclrc-core-scientific-metadata-model.html">CSMD (Core Scientific Metadata Model)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/dif-directory-interchange-format.html">DIF (Directory Interchange Format)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/iso-19115.html">ISO 19115</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/observations-and-measurements.html">Observations and Measurements</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/cerif.html">CERIF (Common European Research Information Format)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/data-package.html">Data Package</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/abcd-access-biological-collection-data.html">ABCD (Access to Biological Collection Data)</a></p>
<p><a href="http://rd-alliance.github.io/metadata-directory/standards/mods.html">MODS (Metadata Object Description Schema)</a></p>
<p><a href="https://github.com/edamontology/edam-bioimaging/blob/master/README.md">EDAM-bioimaging</a></p>
</section>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section"></h2>
</section>
<section id="repositories" class="level2">
<h2 class="anchored" data-anchor-id="repositories">Repositories</h2>
<p><a href="http://www.ddbj.nig.ac.jp/">DNA DataBank of Japan (DDBJ)</a></p>
<p><a href="http://www.ebi.ac.uk/ena/"><strong>European Nucleotide Archive (ENA)</strong></a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/snp">dbSNP</a></p>
<p><a href="https://www.ebi.ac.uk/eva/"><strong>European Variation Archive (EVA)</strong></a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/dbvar/">dbVar</a></p>
<p><a href="http://www.ebi.ac.uk/dgva/">Database of Genomic Variants Archive (DGVa)</a></p>
<p><a href="http://www.ebi.ac.uk/metagenomics">EBI Metagenomics</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/Traces/home/">NCBI Trace Archive</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/sra"><strong>NCBI Sequence Read Archive (SRA)</strong></a></p>
<p><a href="https://www.insdc.org/">International Nucleotide Sequence Database Collaboration</a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/assembly">NCBI Assembly</a></p>
<p><a href="http://data.sbgrid.org/">Structural Biology Data Grid</a></p>
<p><a href="http://wwpdb.org/">Worldwide Protein Data Bank (wwPDB)</a></p>
<p><a href="http://www.uniprot.org">UniProtKB</a></p>
<p><a href="http://www.bmrb.wisc.edu/">Biological Magnetic Resonance Data Bank (BMRB)</a></p>
<p><a href="http://flowrepository.org/">Flowrepository</a></p>
<p><a href="https://www.lipidmaps.org/resources/data">LIPID MAPS</a></p>
<p><a href="https://www.ebi.ac.uk/pride/">PRIDE</a></p>
<p><a href="https://synbiohub.org/">SynBioHub</a></p>
<p><a href="https://openei.org/wiki/Open_Energy_Data_Initiative_(OEDI)">Open Energy Data Initiative (OEDI)</a></p>
<p><a href="https://huggingface.co/datasets">HuggingFace ML datasets</a> (see <a href="https://huggingface.co/blog/researcher-dataset-sharing">blogpost</a> for more info)</p>
</section>
<section id="reporting-standards" class="level2">
<h2 class="anchored" data-anchor-id="reporting-standards">Reporting standards</h2>
<ul>
<li><p><a href="https://doi.org/10.1038/nbt.1823">MIMARKS</a> (and <a href="https://mixs-minas.org/">MINAS</a> for ancient sequences)</p></li>
<li><p><a href="https://reporting-standards.cs.princeton.edu/">REFORMS:</a> Reporting Standards for ML-based Science</p></li>
</ul>
</section>
<section id="rna-seqqpcr" class="level2">
<h2 class="anchored" data-anchor-id="rna-seqqpcr">RNA-seq/qpcr</h2>
<ul>
<li><p><a href="https://www.ebi.ac.uk/ena/browser/home">European Nucleotide Archive</a> (ENA)</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/sra">Sequence Read Archive</a> (SRA)</p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus</a> (GEO)</p></li>
<li><p><a href="https://www.encodeproject.org/">ENCODE</a></p></li>
<li><p><a href="https://exrna-atlas.org/">exRNA Atlas</a></p></li>
<li><p><a href="https://rdml.org/miqe.html">Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE)</a></p></li>
</ul>
</section>
<section id="microscopy" class="level2">
<h2 class="anchored" data-anchor-id="microscopy">Microscopy</h2>
<ul>
<li><p><strong>Repositories</strong></p>
<ul>
<li><p><a href="https://idr.openmicroscopy.org/"><strong>IDR</strong></a>(see <a href="https://idr.openmicroscopy.org/about/faq/">FAQ)</a></p></li>
<li><p><a href="http://www.emdatabank.org/">Electron Microscopy Data Bank (EMDB)</a></p></li>
<li><p><a href="https://www.ebi.ac.uk/pdbe/emdb/empiar/"><strong>EMPIAR</strong></a>(Cryo-EM/ET, 3D EM)</p></li>
<li><p><a href="http://www.cellimagelibrary.org/">Cell Image Library</a></p></li>
<li><p><a href="https://www.ebi.ac.uk/bioimage-archive/">BioImage Archive</a> (2D EM, Light microscopy and other data)</p>
<ul>
<li>Intro video on <a href="https://www.youtube.com/watch?v=nThPvr2gc3U">EMPIAR and BioImage Archive</a></li>
</ul></li>
</ul></li>
<li><p>Image formats</p>
<ul>
<li><p>OME (<a href="https://doi.org/10.1126/science.1082602">Swedlow et al.&nbsp;2003</a>)</p></li>
<li><p>Bio-Formats (<a href="https://doi.org/10.1016/j.ymeth.2015.10.006">Li et al.&nbsp;2016</a>)</p></li>
</ul></li>
<li><p>Metadata</p>
<ul>
<li><p><a href="https://doi.org/10.1038/s41592-021-01166-8">REMBI: Recommended Metadata for Biological Images</a></p></li>
<li><p>light Microscopy Metadata Specifications (<a href="https://doi.org/10.1038/s41592-021-01327-9">Hammer et al.&nbsp;2021</a>)</p></li>
<li><p>3D Microscopy Metadata Standards (3D-MMS) (<a href="https://doi.org/10.1038/s41597-022-01562-5">Ropelewski et al.&nbsp;2021</a>)</p></li>
<li><p>The Minimum Information for High Content Screening Microscopy Experiments (MIHCSME) is a metadata model and reusable tabular template for sharing and integrating high content imaging data. (<a href="https://doi.org/10.1038/s41597-023-02367-w">Hosseini et al.&nbsp;2023</a>)</p></li>
<li><p>Tools such as <a href="https://wu-bimac.github.io/MicroMetaApp.github.io/">Micro-Meta App</a> &amp; <a href="https://github.com/ABIF-McGill/MethodsJ2">MethodsJ2</a></p></li>
</ul></li>
<li><p><a href="https://grussmayerlab.github.io/open-blink/about/">Open blink project</a></p></li>
<li><p><a href="https://github.com/DataCurationNetwork/data-primers/blob/master/Confocal%20Microscopy%20Images%20Data%20Curation%20Primer/confocal-microscopy-images-data-curation-primer.md">Confocal Microscopy Data: A Primer for Curators</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41592-023-01987-9">Community-developed checklists for publishing images and image analyses</a></p></li>
<li><p>Recording analysis steps with ImageJ/Fiji or <a href="https://galaxyproject.org/use/imaging/">Galaxy Imaging</a></p></li>
<li><p>RDM platforms like: <a href="https://www.openmicroscopy.org/omero/">OMERO</a>, <a href="https://cytomine.com/">Cytomine</a>, <a href="https://cyverse.org/bisque-image-analysis">BISQUE</a> and <a href="https://xnat.org">XNAT</a></p></li>
<li><p><a href="https://doi.org/10.1038/s41592-021-01162-y">QUAREP-LiMi: a community endeavor to advance quality assessment and reproducibility in light microscopy</a></p></li>
<li><p>the Digital Imaging and Communications in Medicine <a href="https://www.dicomstandard.org/">(DICOM) standard</a></p></li>
</ul>
</section>
<section id="genomics" class="level2">
<h2 class="anchored" data-anchor-id="genomics">Genomics</h2>
<ul>
<li><p>GenBank, EMBL-Bank, DDBL (processed sequence data), Sequence Read Archive (raw sequencing data)</p></li>
<li><p>Minimum Information about Highly Multiplexed Tissue Imaging (MITI) standard that applies best practices developed for genomics and for other microscopy data to highly multiplexed tissue images and traditional histology. (<a href="https://doi.org/10.1038/s41592-022-01415-4">Schapiro et al.&nbsp;2022</a>)</p></li>
<li><p>MIAME standard for micro array data (<a href="https://doi.org/10.1038/ng1201-365">Brazma et al.&nbsp;2001</a>)</p></li>
<li><p>Minimum information about a genome sequence (MIGS) specification (<a href="https://doi.org/10.1038/nbt1360">Field et al.&nbsp;2008</a>)</p></li>
<li><p>Minimum reporting guidelines for biological and biomedical investigations: the MIBBI project (<a href="https://doi.org/10.1038/nbt.1411">Taylor et al.&nbsp;2008</a>)</p></li>
<li><p><a href="https://doi.org/10.3897/tdwgproceedings.1.20423">Minimum Information Standards (MIxS)</a> established a community-based mechanism for sharing genomic data through a common framework</p></li>
<li><p><a href="https://fairgenomes.org/">FAIR genomes</a> for the medical setting</p></li>
<li><p><a href="https://doi.org/10.5281/zenodo.5706412">MINSEQE</a> describes the Minimum Information about a high-throughput nucleotide SEQuencing Experiment that is needed to enable the unambiguous interpretation and facilitate reproduction of the results of the experiment.</p></li>
<li><p><a href="https://github.com/ebi-gene-expression-group/sc-metadata-fields">MinSCe</a> is a minimum set of single-cell metadata categories and a checklist of information that can be used to describe a single-cell assay in sufficient detail to enable the analysis of transcriptomic data</p></li>
<li><p>Running projects such as: <a href="https://www.ga4gh.org/product/experiments-metadata-standard/" class="uri">https://www.ga4gh.org/product/experiments-metadata-standard/</a></p></li>
<li><p><a href="https://genomicsstandardsconsortium.github.io/mixs/" class="uri">https://genomicsstandardsconsortium.github.io/mixs/</a></p></li>
<li><p><a href="https://ubinfie.github.io/2024/06/06/switch-to-bam.html">From FASTQ to BAM</a></p></li>
</ul>
</section>
<section id="tools" class="level2">
<h2 class="anchored" data-anchor-id="tools">Tools</h2>
<ul>
<li><p><a href="https://www.iochem-bd.org/">ioChem-BD</a></p></li>
<li><p><a href="https://doi.org/10.1016/j.acax.2019.100007">Procedures for systematic capture and management of analytical data in academia</a></p></li>
</ul>
</section>
<section id="simulation-outputs" class="level2">
<h2 class="anchored" data-anchor-id="simulation-outputs">Simulation outputs</h2>
<ul>
<li><p><a href="https://sed-ml.org/">SED-ML</a></p></li>
<li><p><a href="http://biomodels.net/miase">Minimum Information About a Simulation Experiment (MIASE)</a></p></li>
</ul>
</section>
<section id="open-research" class="level2">
<h2 class="anchored" data-anchor-id="open-research">Open Research</h2>
<p><a href="https://osf.io/3r8hb/">Open Research: Examples of good practice, and resources across disciplines</a></p>
</section>
<section id="other" class="level2">
<h2 class="anchored" data-anchor-id="other">Other</h2>
<ul>
<li><a href="https://www.youtube.com/watch?v=o1r4x4qVEBM">Dr.&nbsp;Christoph Steinbeck on “Informatics, Datasets and the Curation of the Future”</a></li>
</ul>
</section>
<section id="mri-data" class="level2">
<h2 class="anchored" data-anchor-id="mri-data">MRI data</h2>
<ul>
<li><p><a href="https://doi.org/10.5281/zenodo.4740126">Brain MRI data sharing guide</a></p></li>
<li><p><a href="https://doi.org/10.1002/hbm.25351">The Open Brain Consent</a></p></li>
</ul>
</section>
<section id="image-data" class="level2">
<h2 class="anchored" data-anchor-id="image-data">Image Data</h2>
<ul>
<li><p><a href="https://central.xnat.org/">XNAT Central</a> (neuroimaging, but also from oncology, orthopaedics and cardiology)</p></li>
<li><p><a href="https://www.cancerimagingarchive.net/">The Cancer Imaging Archive</a></p></li>
</ul>
</section>
<section id="publications-articles" class="level2">
<h2 class="anchored" data-anchor-id="publications-articles">Publications &amp; articles</h2>
<ul>
<li><a href="https://doi.org/10.1016/j.dche.2022.100072">Theisen et al.&nbsp;2023</a></li>
<li><a href="https://www.nature.com/articles/d41586-023-03064-9">How to get microscopes to speak the same language</a></li>
<li><a href="https://doi.org/10.48550/arXiv.1605.04339">Reproducible and replicable CFD: it’s harder than you think</a></li>
</ul>
</section>
<section id="battery" class="level2">
<h2 class="anchored" data-anchor-id="battery">Battery</h2>
<p>[..E]arly independent battery-data activities include those of <a href="https://batteryarchive.org/">Battery Archive</a>, BIG-MAP [(<a href="https://www.big-map.eu/dissemination/battinfo">BattINFO ontology</a>)], <a href="https://batterieseurope.eu/">Batteries Europe,</a> and the <a href="https://www.faraday.ac.uk/">Faraday Institution</a>. Their diversity of locations and formats underscores the critical need for a singular approach to improve uniformity.38,41 - <a href="https://doi.org/10.1016/j.joule.%202022.08.008">Ward et al.&nbsp;2022</a></p>
<ul>
<li><p><a href="https://doi.org/10.1016/j.softx.2020.100506">BEEP</a>: A Python library for Battery Evaluation and Early Prediction</p></li>
<li><p><a href="https://github.com/Battery-Intelligence-Lab/galvanalyser">Galvanalyser</a> is a system for automatically storing data generated by battery cycling machines in a database</p></li>
<li><p><a href="nrel.gov/transportation/microstructure.html">Battery Microstructures Library</a></p></li>
<li><p><a href="https://www.nrel.gov/transportation/matbox.html">MATBOX</a>: Microstructure Analysis Toolbox (Li-ion batteries)</p></li>
<li><p><a href="https://made.ee.ethz.ch/research/open-source-data-and-software/bruggemanestimator.html">BruggemanEstimator</a>: Open Source Code for Estimating the Tortuosity Estimation of Lithium Ion Battery Porous Electrodes from SEM images</p></li>
<li><p><a href="https://github.com/tldr-group/taufactor">TauFactor</a> is an application for calculating tortuosity factors from tomographic data.</p></li>
<li><p><a href="https://github.com/materials-data-facility/battery-data-toolkit">Battery Data Toolkit</a>, converts battery testing data from native formats to a standardized HDF5 file.</p></li>
<li><p><a href="https://doi.org/10.1002/aenm.202102702">Ontologies for battery data</a> (<a href="https://www.big-map.eu/dissemination/battinfo">BattINFO</a> &amp; BVCO)</p>
<ul>
<li>“There are currently two major ongoing initiatives dedicated to ontologizing the battery domain: The Battery Interface Ontology (BattINFO) and the Battery Value Chain Ontology (BVCO). BattINFO describes batteries on the cell level and below, including not only components, materials, and their interfaces, but also electrochemical processes, models, and characterization data. The objective of BattINFO is to support AI workflows and interoperability of battery data in the research and development community. On the other hand, BVCO describes aspects of the battery value chain with a strong focus on battery manufacturing and recycling. Both BattINFO and BVCO stem from the top-level ontology EMMO and are publicly available under open-source licenses.”</li>
</ul></li>
<li><p><a href="https://www.doi.org/10.1002/batt.202100117">BIG-MAP Data Management Plan article</a> + <a href="https://www.big-map.eu/dissemination/public-project-deliverables">deliverable</a></p></li>
<li><p><strong>Repositories:</strong></p>
<ul>
<li><p><a href="https://icsd.products.fiz-karlsruhe.de/">Inorganic Crystal Structure Database</a> (ICSD)</p></li>
<li><p><a href="https://www.materialscloud.org/">Materials Cloud</a></p></li>
<li><p><a href="https://materialsproject.org/">Materials Project</a></p></li>
<li><p><a href="https://cmr.fysik.dtu.dk/">Computational Materials Repository</a></p></li>
</ul></li>
</ul>
</section>
<section id="bionano" class="level2">
<h2 class="anchored" data-anchor-id="bionano">Bio–nano</h2>
<p><a href="https://doi.org/10.1038/s41565-018-0246-4">Minimum information reporting in bio–nano experimental literature</a> (MIRIBEL)</p>
</section>
<section id="cfd" class="level2">
<h2 class="anchored" data-anchor-id="cfd">CFD</h2>
<ul>
<li><p><a href="https://www.youtube.com/watch?v=IXMSOSEj14Q">Machine Learning for Computational Fluid Dynamics</a></p></li>
<li><p>Courses:</p>
<ul>
<li><p><a href="https://www.youtube.com/playlist?list=PL30F4C5ABCE62CB61">ME 702 Computational Fluid Dynamics</a></p></li>
<li><p><a href="https://www.youtube.com/playlist?list=PLvOh309NvHlH0cpKf48VVHr_RUxBF57oY">Computational Fluid Dynamics</a></p></li>
</ul></li>
<li><p><a href="https://www.jousefmurad.com/book-notes/computational-fluid-dynamics-cfd-books/">Books for Computational Fluid Dynamics</a></p></li>
<li><p><a href="https://www.youtube.com/channel/UCcqQi9LT0ETkRoUu8eYaEkg">Fluid Mechanics 101</a></p></li>
<li><p><a href="https://www.youtube.com/watch?v=NlLy-u61yyk">Computational Fluid Dynamics (CFD) - A Beginner’s Guide</a></p></li>
</ul>
</section>
<section id="more-information" class="level2">
<h2 class="anchored" data-anchor-id="more-information">More information</h2>
<ul>
<li><p><a href="https://rdamsc.bath.ac.uk/subject-index">Metadata Standards Catalog</a></p></li>
<li><p><a href="https://fairsharing.org/">FAIRsharing.org</a></p></li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp("https:\/\/EstherPlomp\.github\.io\/TNW-OS-support\/");
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>